### DISCUSSION TITLE
If you hold a glowstick in one hand and a revolver (sword, knife) in the other, then a glowstick will be thrown at the same time as you attack enemy.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13708] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you hold a glowstick in one hand and a revolver (sword, knife) in the other, then a glowstick will be thrown at the same time as you attack enemy.
I think this is not good.
Obviously, the glowstick  is taken in the second hand to highlight the place of the fight.
When attacking with a one-handed weapon, it would be right to continue to hold the glowstick in the second hand.
Another good option will be to make it possible to attach the a glowstick to the head (in place of a helmet).
Currently there is no way to attach a light source to inventory for hands free.
How about to make it possible to hold the glowstick in hand when attacking with second hand?

### Reproduction steps

1. Take the glowstick in one hand and any one-handed weapon in the other.
2. Make an attack with a one-handed weapon.
3. The glowstick is thrown

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.3.0.0/v1.3.0.1

### Which operating system did you encounter this bug on?

Windows

**Comment by ghost** [comment]
> Obviously, the glowstick is taken in the second hand to highlight the place of the fight.

Not... really? Glowsticks are primarily bait for enemies. You should be throwing them instead of holding them.

**Comment by pavel4win** [comment]
If you throw the glowstick  the monster bites it and everything goes dark.
I suggest doing it this way: if the glowstick is in the right hand, then the "attack" throws it. If the glowstick is in the left hand (and weapon in the right), then pressing "attack" has no effect on glowstick . This is logical. People throw things with their right hand. And if they need to illuminate, they hold the light with their left hand and weapon/tool with their right.

**Comment by girlyguppy** [comment]
> Currently there is no way to attach a light source to inventory for hands free.

except where you can place a flashlight (much better light source) on your head

**Reply by pavel4win** [reply]
Of course! Just don't forget to take off your helmet beforehand, so you can get hit on the head in battle.
I meant a diffuse light source. A flashlight gives a directional light source and it is usually directed not at the enemy but anywhere.

**Reply by girlyguppy** [reply]
a glowstick on the head would also prevent you from wearing a helmet, as you previously mentioned
> Another good option will be to make it possible to attach the a glowstick to the head (in place of a helmet).

but yes I agree, I don't see why we can't shove a glowstick up our nose or something, when we can shove a flashlight into our eyesocket.
